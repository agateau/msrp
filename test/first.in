START
ghdsfjkagdsfvasdfjvjkadfk
dsafhadskfjhaskdfjhkjadsfh 
bbbbbbbbbbbbbbbbbbbbb
AAAAAAAAAAAAAAAAA
asdababababba
ABABABBABABABAB
Ab // this should be replaced with two !. the rest should remain unchanged
ABaaabbbbAB Ab aBabababBABBABaBBABABABBABABABbababABABABBBBBbABbABbA
abbabaABBABABb Ab b Ab b Ab bAaB Ab Ab abbabbabbbABBABbB Ab bABbBABbABbABb Ab b Ab
ab
   AB
      Ab
   aB
END