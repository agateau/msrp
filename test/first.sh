#! /bin/sh

echo "TESTING --first option: "

cp -f "$DIR/first.in" tmp.first.out

$MSRP --first -f -d "Ab" "!!" tmp.first.out

cmp --quiet tmp.first.out "$DIR/first.out" || exit 1

echo "PASSED!"
rm tmp.first.out



