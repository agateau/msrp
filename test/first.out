START
ghdsfjkagdsfvasdfjvjkadfk
dsafhadskfjhaskdfjhkjadsfh 
bbbbbbbbbbbbbbbbbbbbb
AAAAAAAAAAAAAAAAA
asdababababba
ABABABBABABABAB
!! // this should be replaced with two !. the rest should remain unchanged
ABaaabbbbAB Ab aBabababBABBABaBBABABABBABABABbababABABABBBBBbABbABbA
abbabaABBABABb Ab b Ab b Ab bAaB Ab Ab abbabbabbbABBABbB Ab bABbBABbABbABb Ab b Ab
ab
   AB
      Ab
   aB
END